src/controllers/login.js

```diff
@@ -49,13 +46,11 @@ const login =
             })
             res.status(200).json({
               message: `${responseAdmin}`,
-              admin: [user.name, user.email],
+              admin: [user.firstname, user.email],
               adminToken
             })
           } else if (
-            bcrypt.compareSync(password, user.password) &&
-            userRole.isOperator
-          ) {
+            bcrypt.compareSync(password, user.password) && user.Role.role === 'operator') {
```

Comment:
I have seen that you have worked on the problem that was causing issues when logging in